repo/open.go

```diff
@@ -450,6 +501,11 @@ func upgradeLockMonitor(
 			return err
 		}
 
+		if err := handleMissingRequiredFeatures(ctx, ufb.repoConfig, false); err != nil {
```

Comment:
Shouldn't `options.TestOnlyIgnoreMissingRequiredFeatures` be respected here as well ?

Reply:
done, although not sure who's gonna see the warnings this way.